tests/integrations_setup/install_azure_asset_inventory_integration.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+"""
+This script installs Asset Inventory Azure integration
+
+The following steps are performed:
+1. Create an agent policy.
+2. Create a CSPM Azure integration.
+"""
+import json
+import sys
+from pathlib import Path
+
+import configuration_fleet as cnfg
+from fleet_api.agent_policy_api import create_agent_policy
+from fleet_api.common_api import (
+    get_arm_template,
+    get_artifact_server,
+    get_enrollment_token,
+    get_fleet_server_host,
+    get_package_version,
+    update_package_version,
+)
+from fleet_api.package_policy_api import create_integration
+from fleet_api.utils import rename_file_by_suffix
+from loguru import logger
+from munch import Munch
+from package_policy import (
+    VERSION_MAP,
+    extract_arm_template_url,
+    generate_random_name,
+    get_package_default_url,
+    load_data,
+    version_compatible,
+)
+from state_file_manager import HostType, PolicyState, state_manager
+
+EXPECTED_AGENTS = 1
+AGENT_POLICY = "../../deploy/cloud/data/agent_policy_asset_inventory_azure.json"
+PACKAGE_POLICY = "../../deploy/cloud/data/package_policy_asset_inventory_azure.json"
+ARM_PARAMETERS = "../../deploy/azure/arm_parameters.json"
+ARM_TEMPLATE = "../../deploy/azure/ARM-for-single-account.json"
+PACKAGE_NAME = "cloud_asset_inventory"
+
+path_agent_policy_data = Path(__file__).parent / AGENT_POLICY
+path_pkg_policy_data = Path(__file__).parent / PACKAGE_POLICY
+path_arm_parameters = Path(__file__).parent / ARM_PARAMETERS
+path_arm_template = Path(__file__).parent / ARM_TEMPLATE
+INTEGRATION_NAME = "Asset Inventory Azure"
+
+PKG_DEFAULT_VERSION = VERSION_MAP.get("asset_inventory_azure", "")
+INTEGRATION_INPUT = {
+    "name": generate_random_name("pkg-asset-inventory-azure"),
+    "input_name": "asset_inventory_azure",
+    "vars": {
+        "azure.account_type": "single-account",
+        "azure.credentials.type": "arm_template",
+    },
+}
+AGENT_INPUT = {
+    "name": generate_random_name("asset-inventory-azure"),
+}
+
+
+if __name__ == "__main__":
+    # pylint: disable=duplicate-code
+    package_version = get_package_version(cfg=cnfg.elk_config, package_name=PACKAGE_NAME)
+    logger.info(f"Package version: {package_version}")
+    if not version_compatible(
+        current_version=package_version,
+        required_version=PKG_DEFAULT_VERSION,
+    ):
+        logger.warning(f"{INTEGRATION_NAME} is not supported in version {package_version}")
```

Comment:
If we are sys.exiting it, feels like a error deserving log 😬

Reply:
Just a copy of what other scripts are doing, but sure, I can bump log level.